rosalind_hamm.py

```diff
@@ -0,0 +1,9 @@
+def main(data):
+    a, b = data[:-1].split('\n')
+    hamm_dist = 0
+    for i in range(len(a)):
+        if a[i] != b[i]:
+            hamm_dist += 1
```

Comment:
I was thinking of something else to make it "right," but I forgot about `sum()`! I'm on it.